src/lib.rs

```diff
@@ -455,7 +451,7 @@ pub fn glob_builder<S: AsRef<str>>(pattern: S) -> GlobWalkerBuilder {
 
         let pat = pattern.to_str().unwrap();
         if cfg!(windows) {
-            GlobWalkerBuilder::new(base.to_str().unwrap(), pat.replace("\\", "/"))
+            GlobWalkerBuilder::new(base.to_str().unwrap(), pat.replace('\\', "/"))
```

Comment:
effectively still the same, clippy suggested replacing it because the regex crate also accepts a single character as pattern